examples/tracing/otlp-collector/.env.example

```diff
@@ -0,0 +1,3 @@
+AZURE_OPENAI_API_KEY=<your_AOAI_key>
+AZURE_OPENAI_API_BASE=<your_AOAI_endpoint>
```

Comment:
Let's put the .env.example in a parent folder, as they can share across examples. 
See this one
https://github.com/microsoft/promptflow/pull/1741/files#diff-b2cfa42efc087d918fe05319dbd27aebbcb516300de250f83a4d7e5508f9c318